frontend/src/index.jsx

```diff
@@ -7,10 +7,12 @@ import HomePage from "./general/HomePage";
 import DeviceListPage from "./devicelist/DeviceListPage";
 import StreamingTablePage from "./createStream/StreamingPage";
 import DeviceDetailsPage from "./deviceDetailsPage/DeviceDetailsPage";
+import LogListPage from "./logslist/LogList";
```

Comment:
Consistency between `LogsList` `LogList` and `LogListPage` for file name vs. import name vs. fodler name
Pick one and stick with it. IMO LogsPage would make most sense, but I leave the decision up to you